Pokrovsky_AsteroidGame/Game.cs

```diff
@@ -64,23 +64,40 @@ public static void Draw()
 
         public static void Load()
         {
-            __GameObjects = new VisualObject[30];
+            int QuantityPlanets = 3;
+            int QuantitiAsetroids = 15;
+            int QuantitiStars = 30;
```

Comment:
Превратить в константы!
```suggestion
            const int QuantityPlanets = 3;
            const int QuantitiAsetroids = 15;
            const int QuantitiStars = 30;
```